sound/soc/intel/boards/Kconfig

```diff
@@ -373,7 +381,7 @@ config SND_SOC_INTEL_GLK_RT5682_MAX98357A_MACH
 
 endif ## SND_SOC_INTEL_SKYLAKE || SND_SOC_SOF_GEMINILAKE
 
-if SND_SOC_INTEL_SKYLAKE || SND_SOC_SOF_CANNONLAKE
+if SND_SOC_INTEL_SKYLAKE || SND_SOC_SOF_CANNONLAKE && SND_SOC_SOF_HDA_LINK
```

Comment:
We don't have HDMI BE dai_links in cnl_rt274.c now so no need this adding?

Reply:
> We don't have HDMI BE dai_links in cnl_rt274.c now so no need this adding?

Ah yes, this was added in a prototype code. Will remove all this anyways.